web/src/app/webview/src/contextManager.ts

```diff
@@ -197,8 +197,8 @@ export default class ContextManager extends ContextManagerBase<WebviewConfigurat
     return activatingKeyboard;
   }
 
-  public resetContext(): void {
-    super.resetContext();
+  public async resetContext(): Promise<void> {
+    await super.resetContext();
     this._rawContext.saveState();
```

Comment:
Currently, we don't `await` this because we aren't dependent on its results to proceed.  If we get predictions back (from the reset) in time, other handling within the engine can handle those predictions.

Per our meeting, we should probably also document the method you were looking to `await` here - to mention the expected use-cases for `await`ing it and when it's fine to... not await.

`this._rawContext.saveState()` - this ensures that the app/webview context's save-state image, which is used for engine / host context synchronization, is current.  We actually _don't_ want to `await` before this.